LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -15,22 +16,36 @@ public class PlaygroundGarush : ISimulation
         List<Guid> subjectIdList = new List<Guid>();
         Dictionary<Guid, Module> moduleDictionary = new Dictionary<Guid, Module>();
         List<Guid> moduleIdList = new List<Guid>();
+        List<Classroom> _classroom = new List<Classroom>();
+
+        public PlaygroundGarush(ReportType reportType)
+        {
+            this.ReportType = reportType;
+        }
+        public ReportType ReportType { get => _reportType; set => _reportType = value; }
+        public List<Classroom> Classroom { get => _classroom; set => _classroom = value; }
```

Comment:
The property's name should be ClassroomList, the field's name should be _classroomList.